### ISSUE TITLE
Abnormal CD8+ T cell subsets

### ISSUE THREAD

**Issue by adamklocperk** [obophenotype/human-phenotype-ontology#11265]
**Preferred term label:**
Abnormal CD8+ T cell subsets

**Synonyms**

**Definition (free text, please give PubMed ID)**
Abnormal increase or decrease of any CD8+ T cell subpopulation, measured as percentage of total CD8+ T cells in the blood, compared to a reference range for a given sex and age-group.

**Parent term (use hpo.jax.org/app)**
Abnormal T cell subset distribution HP:0025540

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-1526-4557

**Comment by pnrobinson** [comment]
@adamklocperk 
Do not understand -- similar question as with: https://github.com/obophenotype/human-phenotype-ontology/issues/11264

**Comment by adamklocperk** [comment]
There need to be four terms as children of Abnormal T cell subset distribution HP:0025540:

[Abnormal CD8+ T cell proportion](https://hpo.jax.org/browse/term/HP:0031393) - denoting that there are more or less total CD8+ T cells as a fraction of total T cells. This term currently exists, but has [Abnormal CD8+ memory T cell proportion](https://hpo.jax.org/browse/term/HP:0410380) and [Abnormal HLA DR+ CD8+ T cell proportion](https://hpo.jax.org/browse/term/HP:0025842) as children, even though these should not be its children, but rather children of Abnormal CD8+ T cell subset proportion (see below).
Abnormal CD4+ T cell proportion - denoting the same, but for CD4+ T cells

[Abnormal CD4+ T cell subset proportion](https://hpo.jax.org/browse/term/HP:0031392) - denoting that the proportion of CD4+ T cells to total T cells may be normal, HOWEVER that WITHIN the CD4+ compartment there are changes (e.g. too many naive CD4+ T cells, too many memory CD4+ T cells, too many RTE CD4+ T cells, too few...etc). This term currently exists, but has two children that don't belong under it, [Increased proportion of CD4-positive T cells](https://hpo.jax.org/browse/term/HP:0032219) and [Decreased proportion of CD4-positive T cells](https://hpo.jax.org/browse/term/HP:0032218). These should be children of Abnormal CD4+ T cell proportion (as suggested above).

Abnormal CD8+ T cell subset proportion - the same, but for CD8+ T cells.

**Comment by pnrobinson** [comment]
@adamklocperk  -- Updated the structure as advised but note there are other entries in the hierarchy. This version will go online this week, we could catch up later if further changes needed.
<img width="569" height="242" alt="Image" src="https://github.com/user-attachments/assets/4255b99a-1fea-485a-bb7a-7fb72e6ba182" />
